### PR TITLE
Consolidate both projects.

### PR DESCRIPTION
**Pull request by sdserranog**
Switch to Biome 
Add vs code recommendations
Fix CSS on mobile
Fix middleware

### CODE REVIEW

File: app/layout.tsx [hunk 1]
```diff
@@ -8,8 +8,8 @@ import './globals.css';
 export const metadata: Metadata = {
   metadataBase: new URL('https://arcade-ai.com'),
   title: 'Arcade AI Chat',
-  description: 'Chat with Archer, an AI assistant built by Arcade AI with access to \
-  service like Google, Spotify, and more.',
+  description:
+    'Chat with Archer, an AI assistant built by Arcade AI with access to service like Google, Spotify, and more.',
```
Comment (nbarbettini) [inline app/layout.tsx:12]: nit: serviceS like...

**Review by nbarbettini — COMMENTED** [review]
One nit

**Review by nbarbettini — APPROVED** [review]
LGTM

**Comment by sdserranog** [comment]
The latest version of this repository can be found in the arcade-chat-app.

**Pull request closed without merging**